metaserver/src/test/scala/tests/compiler/CompletionsTest.scala

```diff
@@ -46,6 +46,7 @@ object CompletionsTest extends CompilerSuite {
       |  "isIncomplete" : false,
       |  "items" : [ {
       |    "label" : "List",
+      |    "kind" : 2,
```

Comment:
this fails. For some reason `List` symbol is detected as `val`

Reply:
This is correct, from the scala package
```scala
  type List[+A] = scala.collection.immutable.List[A]
  val List = scala.collection.immutable.List
```